PinClustering/PinClustering/Model/FuelLocation.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  FuelLocation.swift
+//  PinClustering
+//
+//  Created by Norbert Agoston on 08/09/16.
+//  Copyright © 2016 Norbert Agoston. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+class FuelLocation: NSObject, MKAnnotation {
+    
+    var title: String? {
+        return info?["station_name"] as? String
+    }
+    
+    var subtitle: String? {
+        return info?["fuel_type_code"] as? String
+    }
+    
+    var coordinate: CLLocationCoordinate2D {
+        let locationInfo = info?["location"] as? [String : AnyObject]
+        let latitude = locationInfo?["latitude"] as! String
+        let longitude = locationInfo?["longitude"] as! String
+        
+        return CLLocationCoordinate2D(latitude: Double(latitude)!, longitude: Double(longitude)!)
```

Comment:
i don't like force unwrap. This may crash if you are missing the latitude in the dictionary. Maybe use a default value like this : Double(latitude) ?? 0.0